modules/nw-cluster-mtu-change.adoc

```diff
@@ -347,20 +349,24 @@ where `<config_name>` is the name of the machine config from the `machineconfigu
 If the machine config is successfully deployed, the previous output contains the `/etc/NetworkManager/conf.d/99-<interface>-mtu.conf` file path and the `ExecStart=/usr/local/bin/mtu-migration.sh` line.
 endif::local-zone,wavelength-zone,post-aws-zones,outposts[]
 
-. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
+. Finalize the MTU migration for your plugin. In both example commands, ` <mtu>` specifies the new cluster network MTU that you specified with `<overlay_to>`.
 +
-[source,terminal]
+.. To finalize the MTU migration, enter the following command for the OVN-Kubernetes network plugin:
 +
+[source,terminal]
 ----
 $ oc patch Network.operator.openshift.io cluster --type=merge --patch \
   '{"spec": { "migration": null, "defaultNetwork":{ "ovnKubernetesConfig": { "mtu": <mtu> }}}}'
 ----
 +
---
-where:
-
-`<mtu>`:: Specifies the new cluster network MTU that you specified with `<overlay_to>`.
---
+.. To finalize the MTU migration, enter the following command for the OpenShift SDN network plugin:
```

Comment:
Same here re: unordered list items rather than ordered list items

```suggestion
** To finalize the MTU migration, enter the following command for the OpenShift SDN network plugin:
```

Reply:
Thanks for the great review. Fixed.